### DISCUSSION TITLE
Subdomain whitelist?

### DISCUSSION THREAD

**Discussion by megamorphg** [darkreader/darkreader#8176] [Q&A]
Is it possible to whitelist a subdomain (while still blacklisting the main domain) currently in the latest preview (4.9.45)?

This is in reference to this:
https://github.com/darkreader/darkreader/issues/147

I really need this because there are some domains I use for work with literally dozens of subdomains.
I'd like to blacklist the domain as a whole and then there are about a dozen subdomains that I would then need to whitelist (within that domain).

(I don't want to change my global option to invert because I do not want that globally, just need to whitelist for certain subdomains.)

**Comment by KaKi87** [comment]
#147 will be fixed by #2517, allowing 2nd-level domains to be excluded without excluding lower-level domains.

In the meantime, my [Dark Reader dynamic blacklist](https://git.kaki87.net/KaKi87/userscripts/src/branch/master/darkReaderDynamicBlacklist/README.md) userscript may be useful, if you're using Dark Reader in blacklist mode (I didn't quite understand whether you're in blacklist mode or whitelist mode).
